docs/conf.py

```diff
@@ -41,3 +42,10 @@
     ('index', 'icalendar', 'icalendar Documentation',
      ['Plone Foundation'], 1)
 ]
+
+intersphinx_mapping = {
+    "python": ("https://docs.python.org/3", None),
+    "dateutil": ("https://dateutil.readthedocs.io/en/stable/", None),
+    "tzdata": ("https://tzdata.readthedocs.io/en/latest/", None),
+    "pytz": ("https://pythonhosted.org/pytz/", None),
```

Comment:
I don't think there are any linked references to dateutil, tzdata, or pytz. I would remove them, else it forces unnecessary downloads of the `obj.inv` files on which Intersphinx depends, slowing down build times and consuming electricity.

Reply:
Yes, you are welcome to remove them!